dist/Devel-PPPort/t/ppphtest.t

```diff
@@ -721,7 +721,7 @@ my %p;
 my $fail = 0;
 for (@o) {
   my($name, $flags) = /^(\w+)(?:\s+\[(\w+(?:,\s+\w+)*)\])?$/ or $fail++;
-  exists $p{$name} and $fail++;
+  { exists $p{$name} and $fail++; }
```

Comment:
Why is this extra scope required?

Reply:
This is kinda a conversation for the other repo. The code comes from here. https://github.com/Dual-Life/Devel-PPPort/commit/8b9ff6d9e9fcb53ca615b72e5a5ed60a8a959bb7

Reply:
A side conversation to be had is that D-PPP is currently  being developed like it is in cpan/

Reply:
It seems to be leftover from the removal of `BEGIN` block inside that block...
If you think this is an issue I would suggest to bring the discussion / open a case in D-PPP itself.
@khwilliamson any feedbacks?
IMO this extra block, even if useless does not hurt, I can submit a PR to D-PPP to remove it
closing the discussion here

Reply:
As a single change, this looks weird and confusing, certainly because the surrounding lines do not use an extra scope for the same construct.
As Devel::PPPort is hard enough to maintain, I'd indeed like to see the code as non-complex as possible and drop the extra scope, but I would not consider it a blocker.

Reply:
this will be solved in the next D-PPP update